assets/app/views/browse/build-config.html

```diff
@@ -164,6 +164,7 @@
               <span ng-if="!(build | annotation : 'buildNumber')">
                 <a ng-href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfig.metadata.name}}/{{build.metadata.name}}">{{build.metadata.name}}</a>
               </span>
+              <span ng-if="build.status.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{build.status.message}}"></span>
```

Comment:
because @smarterclayton  didn't like the long messages on the black tooltip background.   And i tend to agree, tooltip is fine for short messages but the longer ones are a bit hard to read